pkg/controller/servicemesh/controlplane/reconciler.go

```diff
@@ -79,11 +82,11 @@ func NewControlPlaneInstanceReconciler(controllerResources common.ControllerReso
 	}
 }
 
-func (r *controlPlaneInstanceReconciler) Reconcile() (result reconcile.Result, err error) {
+func (r *controlPlaneInstanceReconciler) Reconcile(ctx context.Context) (result reconcile.Result, err error) {
```

Comment:
Should we be creating a new child context here?

Reply:
I don't see a reason to create one. Currently, we only create a new Context when we add fields the logger. I can imagine we'll want to create new Contexts when we want to be able to cancel them. But that's not the case yet.